### DISCUSSION TITLE
New contribution proposal for CDT Cloud. Trace-event-logger

### DISCUSSION THREAD

**Discussion by MatthewKhouzam** [eclipse-cdt-cloud/cdt-cloud#37] [Show and tell]
Hi all,

We have several Java based applications that need timing logging. We have already instrumented a few and would like to propose a standalone library rather than re-implementing the wheel each time.

The proposed solution is here.

https://github.com/tracecompass/trace-event-logger

This is the initial logger. It needs a few things.

1. Become a real eclipse project, gonna propose to cdt-cloud
2. Add a build action to run unit tests
3. Publish jars to somewhere like maven central

I feel that this would be a good fit in CDT-cloud. This should be a fairly low activity project as the code was stable for years beforehand. Any thoughts/suggestions?

**Comment by MatthewKhouzam** [comment]
@planger @thegecko I would like your take on this as it would affect projects like OpenVSX, I hope.

**Reply by planger** [reply]
I'm fine with hosting this in CDT Cloud, if it is used by or going to be used by a CDT Cloud component. On its own, I'm not sure if it has a clear relationship to CDT-related topics. Wouldn't it be a great fit for being hosted in Eclipse TraceCompass itself?
Thanks!

**Reply by MatthewKhouzam** [reply]
You have a very good point, I honestly forgot the trace compass server was part of tools and not cdt cloud. :) It should be here https://github.com/eclipse-tracecompass. Thanks!
